pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -184,6 +185,17 @@ var adminCommands = []*cli.Command{
 			},
 		},
 	},
+	{
+		Name:  "secrets-consolidation",
+		Usage: "Runs a script that consolidates secrets in your database",
```

Comment:
I think it could use a move verbose but clearer explanation of what it does: re-encrypts all encrypted values with new data keys and delete the old/deactivated data keys.

Reply:
since we are on this subject, maybe change script -> operation?